test/ng/directive/ngModelSpec.js

```diff
@@ -1368,6 +1328,26 @@ describe('ngModel', function() {
   describe('CSS classes', function() {
     var EMAIL_REGEXP = /^[a-z0-9!#$%&'*+\/=?^_`{|}~.-]+@[a-z0-9]([a-z0-9-]*[a-z0-9])?(\.[a-z0-9]([a-z0-9-]*[a-z0-9])?)*$/i;
 
+    it('should set ng-empty or ng-not-empty the view value changes', inject(function($compile, $rootScope, $sniffer) {
```

Comment:
Theoretically, yes, they should be 100 chars max :smiley:
See: [CONTRIBUTING.md#-coding-rules][1]

[1]: https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#-coding-rules